inference-contract/docs/notebooks/documents/overview.md

```diff
@@ -0,0 +1,61 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Inference contracts using PDO #
+
+The Inference contract family is a suite of contracts that extend the
+capabilities of the [Exchange Contract Family ](../exchange/documents/overview.ipynb)
+in order to create a confidentiality preserving policy-wrapper around the
+usage of a machine learning (ML) model. At its core, the implementation
+contains a Token Object PDO contract that specifies and checks the
+policies to be followed while using the ML model for inferencing
+operations. In this implementation, the ML Model is assumed to be an
+[OpenVINO based IR](https://docs.openvino.ai/2023.2/openvino_ir.html)
+model. The model itself is deployed via an asset guardian service; the
+overall architecture of how a PDO token object gets used to manage an
+asset deployed via an asset guardian service is largely similar to the
+one used within the [Digital Asset Contract Family](https://github.com/hyperledger-labs/pdo-contracts/tree/main/digital-asset-contract). Note that the
+digital asset contract family in turn relies on the
+[Exchange Contract Family ](../exchange/documents/overview.ipynb)
+for the foundational secure communication protocols between the Token
+Object and the Asset Guardian Service.
+
+The asset guardian service here is implemented as a web service
+containing a frontend and backend portions.The backend of the guardian
+service implements the OpenVINO model server capable of performing
+inferencing operations using the OpenVINO IR model. The frontend of
+the guardian service at a high level performs two
+functions: 1. Processes the capability package generated by the token
+object and generates/invokes the API for the backend. The capability
+package may be thought of the asset-usage approval generated by the
+token object smart contract. 2. Implements any input preprocessing and
+output postprocessing (collectively referred to as model scoring) as
+determined necessary by the original asset owner.
```

Comment:
This will ensure these are rendered as numbered bullets.

```suggestion
functions: 

1. Processes the capability package generated by the token
object and generates/invokes the API for the backend. The capability
package may be thought of the asset-usage approval generated by the
token object smart contract. 

2. Implements any input preprocessing and
output postprocessing (collectively referred to as model scoring) as
determined necessary by the original asset owner.
```

Reply:
thanks, done